packages/react-storage/src/components/FileUploader/ui/FileList/types.ts

```diff
@@ -0,0 +1,59 @@
+import { StorageManagerDisplayTextDefault, TaskHandler } from '../../utils';
+import { FileStatus, StorageFile } from '../../types';
+
+export interface FileListProps {
+  displayText: StorageManagerDisplayTextDefault;
+  files: StorageFile[];
+  isResumable: boolean;
+  onCancelUpload: TaskHandler;
+  onDeleteUpload: (params: { id: string }) => void;
+  onPause: TaskHandler;
+  onResume: TaskHandler;
+  showThumbnails: boolean;
+  hasMaxFilesError: boolean;
+  maxFileCount: number;
+}
+
+export interface FileControlProps {
+  displayText: StorageManagerDisplayTextDefault;
+  displayName: string;
+  errorMessage: string;
+  isImage: boolean;
+  isResumable: boolean;
+  isUploading: boolean;
+  loaderIsDeterminate: boolean;
+  onRemove: () => void;
+  onPause: () => void;
+  onResume: () => void;
+  progress: number;
+  showThumbnails: boolean;
+  size?: number;
+  status: FileStatus;
+  thumbnailUrl: string;
+}
+
+export interface FileStatusMessageProps
+  extends Pick<
+    StorageManagerDisplayTextDefault,
```

Comment:
```suggestion
    FileUploaderDisplayTextDefault,
```

Reply:
Updated to use name referencing FileUploader